core/dbt/utils.py

```diff
@@ -298,7 +298,7 @@ def filter_null_values(input: Dict[K_T, Optional[V_T]]) -> Dict[K_T, V_T]:
 
 
 def add_ephemeral_model_prefix(s: str) -> str:
-    return '__dbt__CTE__{}'.format(s)
+    return '__dbt__cte__{}'.format(s)
```

Comment:
For clarity, this method is no longer used, since #2712 swapped it out in favor of [`adapter.add_ephemeral_prefix`](https://github.com/fishtown-analytics/dbt/pull/3028/files#diff-8b301fa58dbc12c519f4bf6e3056605c09aea9da46b21d2ec5fd979cfd4f38adR205).